src/lib/common/sem.cpp

```diff
@@ -504,8 +622,10 @@ static int get_curl_context_reuse(const std::string& key, struct curl_context* p
   if (s != 0)
   {
     LM_E(("Runtime Error (pthread_mutex_lock failure)"));
+    ++contexts_mutex_errors;
```

Comment:
I'd suggest to add a FIXME after its declaration to explaint that it is not yet used.